modules/ROOT/pages/migration-transports.adoc

```diff
@@ -9,6 +9,51 @@ replaceable by a connector in Mule 4.
 This page addresses items of migration that are common to all transports. For transport-specific migration
 documentation, refer to xref:migration-connectors.adoc[Migrating Connectors and Modules to Mule 4] and its child pages.
 
+[[address]]
+== Endpoint Addresses
+
+In Mule 3, an endpoint could be configured with the attributes that are specific to the transport (for instance, `host`, `port` and `path` for http) or with the generic `address` attribute.
+
+In the case where the generic `address` attribute was used, it was possible to reference a property, for instance:
```

Comment:
I'd like to suggest a small edit to these two lines:
> In Mule 3, an endpoint could be configured with either transport-specific attributes (for example, an HTTP endpoint could use `host`, `port`, and `path`), or with the generic `address` attribute.
Whenever the generic `address` attribute was used, it was possible to reference a property. For instance: